qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User based login
+
+@selenium
+@defaultLogin
+Scenario Outline: User Based login verification
```

Comment:
When do we use `Scenario Outline` over `Scenario` ?

Reply:
Probably we can use scenario since we got few outline tests.